packages/cli/package.json

```diff
@@ -50,7 +50,7 @@
     "cli-table3": "0.6.5",
     "colors": "1.4.0",
     "debug": "4.4.1",
-    "decompress": "4.2.1",
+    "decompress-unzip": "4.0.1",
```

Comment:
Note: It appears we're replacing only one dependency. But in fact, the number of dependencies is cut.

The original dependency tree looks like:

```
zapier-platform-cli
└ decompress
  │ decompress-tar <- this is the one we want to get rid of
  │ decompress-targz2
  │ decompress-targz
  └ decompress-unzip
    └ yauzl
```

The new dependency tree will look like:

```
zapier-platform-cli
└ decompress-unzip
  └ yauzl
```